main.go

```diff
@@ -115,21 +115,19 @@ func handleSigterm(stopChan chan struct{}) {
 	close(stopChan)
 }
 
-func newClient() (*kubernetes.Clientset, error) {
+func newClient(cfg *config.Config) (*kubernetes.Clientset, error) {
 	var (
 		config *rest.Config
 		err    error
 	)
 
-	kubeconfig := clientcmd.RecommendedHomeFile
-
-	// if inCluster {
-	// config, err = rest.InClusterConfig()
-	// log.Debug("Using in-cluster config.")
-	// } else {
-	config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
-	log.Debugf("Using current context from kubeconfig at %s.", kubeconfig)
-	// }
+	if cfg.InCluster {
+		config, err = rest.InClusterConfig()
+		log.Debug("Using in-cluster config.")
+	} else {
+		config, err = clientcmd.BuildConfigFromFlags("", cfg.KubeConfig)
```

Comment:
alright, changed, but with one twist. I left the `in-cluster` at `false` by default, so it's easier to use locally for now.